lib/webauthn/authenticator_attestation_response.rb

```diff
@@ -19,7 +19,7 @@ def initialize(attestation_object:, **options)
       @attestation_object = attestation_object
     end
 
-    def valid?(original_challenge, original_origin, rp_id: nil)
+    def verify(original_challenge, original_origin, rp_id: nil)
```

Comment:
I think this method can be simplified after you refactor.

The `return false unless valid_response` becomes dead code as it won't never `return false` if the `super` call raises.

It would also be great if `attestation_statement.valid?` in this method could also do have a `or WebAuthn::AttestationStatementError` and remove the guard clause below it. Not suggesting to add to much scope and add error handling yet inside the Attestation Statement verification.

Reply:
Good one!
Nice, thanks.